cfme/fixtures/authentication.py

```diff
@@ -74,3 +74,4 @@ def configure_auth(appliance, auth_mode, auth_provider, user_type, request, fix_
     appliance.server.authentication.auth_settings = original_config
     appliance.evmserverd.restart()
     appliance.wait_for_web_ui()
+    appliance.server.logout()
```

Comment:
Another options here is if you call 'appliance.server.browser.refresh`, you will be redirected to the logout page as well.

**Required** Please add a comment here explaining why you are logging out so it is clear for the next engineer that looks at this fixture.

Reply:
Comment added.